persistence-jdbc/scaladsl/src/test/resources/reference.conf

```diff
@@ -1 +1,14 @@
 jdbc-defaults.slick.profile = "slick.jdbc.H2Profile$"
+
+
+# Aggressively restart and ensure the cluster-distributed, readside actors so tests don't take that long.
+lagom.persistence {
+  read-side.failure-exponential-backoff {
+    min = 1 s
+    max = 2 s
+    random-factor = 0.2
+  }
+  cluster.distribution {
+    ensure-active-interval = 4s
+  }
+}
```

Comment:
Non multi-JVM tests for JDBC persistence suffer the same flakyness multi-JVM tests do (see #991).